Python/sysmodule.c

```diff
@@ -2421,6 +2421,114 @@ sys_is_stack_trampoline_active_impl(PyObject *module)
     Py_RETURN_FALSE;
 }
 
+
+/*[clinic input]
+sys.is_remote_debug_enabled
+
+Return True if remote debugging is enabled, False otherwise.
+
+If no stack profiler is activated, this function has no effect.
+[clinic start generated code]*/
+
+static PyObject *
+sys_is_remote_debug_enabled_impl(PyObject *module)
+/*[clinic end generated code: output=7ca3d38bdd5935eb input=c8b9354cb7a08ed6]*/
+{
+#ifndef Py_REMOTE_DEBUG
+    Py_RETURN_FALSE;
+#else
+    const PyConfig *config = _Py_GetConfig();
+    return PyBool_FromLong(config->remote_debug);
+#endif
+}
+
+/*[clinic input]
+sys.remote_exec
+
+    pid: int
+    script: object
+
+Executes a file containing Python code in a given remote Python process.
+
+This function returns immediately, and the code will be executed by the
+target process's main thread at the next available opportunity, similarly
+to how signals are handled. There is no interface to determine when the
+code has been executed. The caller is responsible for making sure that
+the file still exists whenever the remote process tries to read it and that
+it hasn't been overwritten.
+
+Args:
+     pid (int): The process ID of the target Python process.
+     script (str|bytes): The path to a file containing
+         the Python code to be executed.
+[clinic start generated code]*/
+
+static PyObject *
+sys_remote_exec_impl(PyObject *module, int pid, PyObject *script)
+/*[clinic end generated code: output=7d94c56afe4a52c0 input=5749b0253d5b588c]*/
+{
+#ifdef MS_WINDOWS
+    // Get UTF-16 (wide char) version of the path for Windows
+    wchar_t *debugger_script_path_w = PyUnicode_AsWideCharString(script, NULL);
```

Comment:
Above we said that `script` could be either `str|bytes`, but I believe `PyUnicode_AsWideCharString` only accepts `str`. Perhaps on Windows the `bytes` option doesn't even make sense, though. Should we document that Unix allows `bytes` but Windows does not?